message MonitorInfo{
    required string monitorID=1;
    required string hostname=2;
    optional int32 port =8 [default = 5051];
}

